template-examples/README.md

```diff
@@ -0,0 +1,12 @@
+This subdirectory contains a few examples of how to use the Coq-community [template](https://github.com/coq-community/templates).
+
+The files here can be generated in the following way, assuming `t` is set to the template directory:
+
+``` shell
+for dir in aac-tactics chapar lemma-overloading; do
+  for f in $t/default.nix.mustache $t/README.md.mustache $t/.travis.yml.mustache; do
+    mustache $dir/meta.yml $f > $dir/${f%.mustache} && echo "$dir/${f%.mustache}"
+  do
+  mustache $dir/meta.yml $t/coq.opam.mustache > $dir/coq-$dir.opam && echo "$dir/coq-$dir.opam"
+done
+```
```

Comment:
I don't like that this is repeated between the README of the templates repository and here, because this is easy to get them out of sync. I would prefer for the templates repository to contain the generation process, e.g. as a shell script, and for this README to refer to it.

Reply:
I've added `templates` as submodule, and a `Makefile` to illustrate the usages.